### ISSUE TITLE
bower install angular-formly fails with Package apiCheck not found

### ISSUE THREAD

**Issue by rbfinch** [formly-js/angular-formly#193]
Trying to install angular-formly fails:

```
$ bower install angular-formly --save
bower cached        git://github.com/formly-js/angular-formly.git#4.0.2
bower validate      4.0.2 against git://github.com/formly-js/angular-formly.git#*
bower ENOTFOUND     Package apiCheck not found
```
It appears that apiCheck was renamed to api-check but angular-formly's bower.json still has the old name.

**Comment by kentcdodds** [comment]
Thanks for letting me know. Sorry about that!

**Comment by kentcdodds** [comment]
It's fixed in 4.0.3
